test/elixir/test/bulk_docs_test.exs

```diff
@@ -129,6 +129,13 @@ defmodule BulkDocsTest do
     assert Enum.at(rows, 2)["error"] == "conflict"
   end
 
+  @tag :with_db
+  test "bulk docs raises transaction_too_large error for transaction larger than 10MB", ctx do
+    docs = [%{_id: "0", a: random_string(16_000_000)}]
+    resp = Couch.post("/#{ctx[:db_name]}/_bulk_docs", body: %{docs: docs})
+    assert resp.status_code == 413
```

Comment:
Let's match on the error message ("transaction_too_large") since we're testing for it.

Wondering if with 16MB we would hit transaction_too_large or the document size limit of 8MB. If we don't hit 8MB we may have another bug to fix about why that doesn't work.

Reply:
There are different settings on CouchDB and Cloudant for Transaction Engine for `max_document_size`.  

```
max_document_size = 8000000 ; bytes  (CouchDB)
max_document_size = 67108864 ; 64 MB (Cloudant for Transaction Engine)
```

So by default, we will get `document_too_large` in CouchDB while we will get `transaction_too_large` if we pass 16MB document. 

I added codes to adjust `max_document_size` in this test to support 64MB documents to get `transaction_too_large` in any cases.

Reply:
@jiangphcn 

Good find!

TxE is technically Cloudant's problem and shouldn't have any relevance for CouchDB. Though looking at the code there, is that the /dev/run instance may have 65Mb however the instance used in production has 1Mb.

Also maybe in a separate commit, we could probably switch to using the code default (in Erlang) to be 8Mb instead of keeping in the default.ini. Then default.ini can be a commented out line with the default value in it like we usually do it.